docs/motivation.md

```diff
@@ -0,0 +1,91 @@
+# Motivation
+
+*Jeff* was designed to solve a few very specific problems the field is currently encountering.
+
+## Better interdisciplinary support.
+
+**We need to be able to compile large, complex, quantum algorithms, and this is an interdisciplinary
+effort.**
+
+Compilation is an incredibly important component of quantum computing infrastructure. The inability
+to optimize large algorithms for future fault-tolerant hardware will extend (or block!) timelines
+for utility-scale quantum computation.
+
+However, there is still a lot of work to do, and a lot of uncertainty. We need to:
+
+- continue to make new theoretical compilation breakthroughs; 
+- identify the most-promising large scale algorithms; and finally,
+- figure out how to make this practical through quantum software.
+  
+To do so, we need to bridge the gap between researchers doing fundamental compilation and
+algorithmic research, and quantum software developers.
+
+- **How can we share results and ideas faster, within a common framework?**
+- **How do we make sure we are speaking a common language, and aware of challenges and approaches
+    on both sides?**
```

Comment:
These feel like they stand out here for no reason. Would it be worth adding lead-in or lead-out from the questions?